samples/bootloader/pm.yml

```diff
@@ -1,5 +1,5 @@
 #include <autoconf.h>
-#include <devicetree_unfixed.h>
+#include <devicetree_legacy_unfixed.h>
```

Comment:
done

Reply:
unfortunately I had to go back to the previous version. Introducing the change is not so trivial because it requires c-lib preprocessing for the PM. This can be tweaked after the merge,

Reply:
ACK